docs/state.md

```diff
@@ -40,25 +40,29 @@ The worker maintains a worker state which is a JSON representation of the follow
 
 ```jsonc
 {
-    "worker_id": "<WORKER_ID>"
+    "worker_id": "<WORKER_ID>",
+    "instance_id": "<INSTANCE_ID>"  # OPTIONAL
 }
 ```
 
 #### `worker_id`
 
-This contains a unique identifier that represents the worker in your AWS Deadline Cloud fleet.
+This contains a unique identifier that represents the worker in your AWS Deadline Cloud fleet, as 
+well as an optional identifier of the EC2 instance running the worker, if applicable. 
```

Comment:
Sorry, I must have missed this in my original review....

This documentation for `instance_id` is under the `worker_id` section. Can we make a new one for `instance_id`?